web/docs/auth/auth-hooks.md

```diff
@@ -679,6 +772,50 @@ The hook receives an object as **input** with the following properties:
 
 Wasp ignores this hook's **return value**.
 
+### The `onAfterEmailVerified` hook
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```js title="src/auth/hooks.js"
+export const onAfterEmailVerified = async ({ email, user, prisma, req }) => {
+  // Hook code goes here
+}
+```
+
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```ts title="src/auth/hooks.ts"
+import type { OnAfterEmailVerifiedHook } from 'wasp/server/auth'
+
+export const onAfterEmailVerified: OnAfterEmailVerifiedHook = async ({
+  email,
+  user,
+  prisma,
+  req,
+}) => {
+  // Hook code goes here
+}
+```
+
+</TabItem>
+</Tabs>
+
+The hook receives an object as **input** with the following properties:
+
+- `email: string`
+
+  The email address that was verified.
```

Comment:
Passive voice again :smile: 

Difficult to avoid with a full sentence, but the good news is we don't need it. We can say:
> The user's veriried email address.